src/CodiceFiscale/Calculator.php

```diff
@@ -65,7 +67,7 @@ public function calculateAllPossibilities()
     /**
      * Calculate the surname part of the codice fiscale.
      *
-     * @returns Returns the surname part of the codice fiscale.
+     * @return string Returns the surname part of the codice fiscale.
```

Comment:
You could remove **Returns** here and use an uppercase **T** for the initial **the**.